server/utils/get_request_data_csv.py

```diff
@@ -1,11 +1,11 @@
-import pandas as pd
+import pandas 

```

Comment:
So I was suggesting to order them as such:

```
import argparse
import requests

import pandas as pd
```

Basically, we want to import things in this order:
1. Standard library imports.
2. Related third party imports.

Within each of those, we sort the imports by alphabetical order.

Reply:
Thanks Nich for explaining it clearly. Incorporated the change.